src/test/resources/Features/OCD-1764-chpldownload.feature

```diff
@@ -10,3 +9,10 @@ Feature: OCD-1764 - Chpl data files are downloadable and have respective definit
     Given I am on download the CHPL resources page on "DEV"
     When user selects a file in download file box
     Then definition file shows based on download file selection
+
+  Scenario: Admin user sees eight download files
+    Given I'm logged in as an Admin
+    And I should be logged in to CHPL as "Admin"
```

Comment:
This `And` step should not be here. This step is a validation step and not appropriate for this scenario